Mapping.c

```diff
@@ -311,3 +312,69 @@ const cs_ac_type mapping_get_op_access(MCInst *MI, unsigned OpNum,
 DEFINE_get_detail_op(arm, ARM);
 DEFINE_get_detail_op(ppc, PPC);
 DEFINE_get_detail_op(tricore, TriCore);
+
+/// Returns true if for this architecture the
+/// alias operands should be filled.
+/// TODO: Replace this with a proper option.
```

Comment:
Do we have an issue for this?

Reply:
Not yet. I need to update `AArch64` and `ARM` for it as well. Do this after or just before everything is updated to LLVM 17.